component/class/defaults.yml

```diff
@@ -15,5 +15,12 @@ parameters:
         repository: 'vshn/exoscale-metrics-collector'
         tag: 'v0.3.0'
     # Times in UTC! Don't run job around midnight as exoscale API may return incomplete data
+
+    # schedule for objectstorage cronjob
     # default: Every day at minute 10 past hour 10, 16 and 20.
     schedule: '10 10,16,20 * * *'
+
+    # enable DBaaS cronjob in addition to objectstorage cronjob.
+    dbaasEnabled: false
+    # schedule for DBaaS cronjob every 15min
+    dbaasSchedule: '*/15 * * * *'
```

Comment:
Nitpick/Personal Opinion. I'm usually a fan of nested keys for something like this. So something like:

```
dbaas:
    enabled: false
    schedule: '*/15 * * * *''
```

But it's also fine as is

Reply:
:+1: